### PR TITLE
FRG3D-7 upload form into main

### PR DESCRIPTION
**Pull request by charleslu0529**
Upload form created.

Current functionality: Save file into browser storage and generate a URL link to the file.

TODO: Wrap form inside the modal component once modal component is available.

Ticket: https://jaychungart.atlassian.net/browse/FRG3D-7

### CODE REVIEW

File: src/Components/UploadForm/UploadForm.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        }
+    };
+
+    return (
+        <form
+            action=""
+            className="upload-form"
+            onSubmit={(event) => uploadFile(event)}
+        >
+            <h2 htmlFor="">Upload 3D model</h2>
```
Comment (REACH47) [inline src/Components/UploadForm/UploadForm.jsx]: Is it good practice to leave this 'htmlFor' empty even though it's not in use?

**Review by REACH47 — COMMENTED** [review]
Incredibly detailed and great code quality/readability.
Excellent work as usual Charles.

File: src/Components/UploadForm/UploadForm.jsx (hunk 1 above)
Comment (charleslu0529) [reply src/Components/UploadForm/UploadForm.jsx]: Good catch. This shouldn't actually be needed for the h2 tag. I will change it and push a new commit

**Comment by REACH47** [comment]
Approved. Merging and closing.

**Pull request merged**